scanners/flutter/flutter.go

```diff
@@ -249,74 +244,35 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
 	flutterProjectLocationOption := models.NewOption(projectLocationInputTitle, projectLocationInputSummary, projectLocationInputEnvKey, models.TypeSelector)
 
 	for _, project := range scanner.projects {
+		var testKey string
 		if project.hasTest {
-			flutterProjectHasTestOption := models.NewOption(testsInputTitle, testsInputSummary, "", models.TypeSelector)
-			flutterProjectLocationOption.AddOption(project.path, flutterProjectHasTestOption)
-
-			for _, v := range []string{"yes", "no"} {
-				cfg := configName
-				if v == "yes" {
-					cfg += "-test"
-				}
+			testKey = "test"
+		} else {
+			testKey = "notest"
+		}
+		cfg := configName + "-" + testKey
 
-				if project.hasIosProject || project.hasAndroidProject {
-					if project.hasIosProject {
```

Comment:
This nested if was unnecessary complex. The changed lines below are caused by the indent change